src/app/accounts/login.component.html

```diff
@@ -1,10 +1,10 @@
 <form id="login-page" #form (ngSubmit)="submit()" [appApiAction]="localFormPromise" attr.aria-hidden="{{showingModal}}">
-    <div class="content CozyTheme--inverted">
+    <div class="content">
```

Comment:
Did the rules changed about the inverted theme? The rule was to use it everywhere we have to request elevation rights (i.e. login, vault unlock, authorization screen etc)

Reply:
After confirming with design team, yes, no more UI differences when elevating rights.